test_helpers/tcs/tcs.go

```diff
@@ -0,0 +1,98 @@
+package tcs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+type Tcs struct {
+	// conn Connection interface for interacting with Tarantool.
+	doer tarantool.Doer
+}
+
+// GetResponse is a response of a get request.
+type GetResponse struct {
+	Data []struct {
+		Path        string
+		Value       string
+		ModRevision int64 `mapstructure:"mod_revision"`
+	}
+}
+
+// StartTcs starts a Tarantool centralized configuration storage.
+// Returns a Tcs instance and a cleanup function.
+func StartTcs(opts test_helpers.StartOpts) (Tcs, func()) {
+	if less, err := test_helpers.IsTarantoolEEVersionLess(3, 3, 0); less || err != nil {
+		log.Fatal("Tarantool EE 3.3+ required", err)
```

Comment:
```suggestion
func StartTCS(t testing.TB, opts test_helpers.StartOpts) (Tcs, func()) {
        t.Helper()

	if less, err := test_helpers.IsTarantoolEEVersionLess(3, 3, 0); less || err != nil {
		t.Fatal("Tarantool EE 3.3+ required", err)
```

Reply:
> But the same helpers used for `Example` methods, where no any `testing.TB` arguments.

Probably, we could create two different start helpers:

```
Start() (tcs.Tcs, error)
```

And

```
StartTesting(t testing.TB) tcs.Tcs
```

So in unit-tests it will be:

```
tcs := StartTesting(t) // And handle errors with t.Fatal() inside.
```

For examples:

```
tcs, err := Start()
if err != nil {
    log.Fatal(err)
}
```